pkg/services/oracle/neofs/neofs.go

```diff
@@ -54,7 +54,6 @@ func Get(ctx context.Context, priv *keys.PrivateKey, u *url.URL, addr string) (i
 		prmi client.PrmInit
 		c    *client.Client
 	)
-	prmi.SetDefaultSigner(neofsecdsa.Signer(priv.PrivateKey))
 	c, err = client.New(prmi)
```

Comment:
```suggestion
	c, err := client.New(client.PrmInit{})
```
and remove this `var` decl above then?

Reply:
So be it.